zetaclient/chains/ton/liteapi/client_live_test.go

```diff
@@ -8,12 +8,15 @@ import (
 	"testing"
 	"time"
 
+	"cosmossdk.io/math"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/tonkeeper/tongo/config"
 	"github.com/tonkeeper/tongo/liteapi"
 	"github.com/tonkeeper/tongo/tlb"
 	"github.com/tonkeeper/tongo/ton"
+	toncontracts "github.com/zeta-chain/node/pkg/contracts/ton"
```

Comment:
```suggestion

	toncontracts "github.com/zeta-chain/node/pkg/contracts/ton"
```
Weird I thought generate would fail we don't have a space between external and local packages

Reply:
wdym? I don't see an issue here. https://github.com/zeta-chain/node/blob/f0ed61d88286a3b0aff9d69b0e26e685541eb458/.golangci.yml#L44